### PR TITLE
added Gymnasium button 

### PR DESCRIPTION
**Pull request by nidhinmahesh**
I have mapped from local development repo to development repo of org. Please review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ title: Susi - Docs
 
 ---
 # Susi
+[![Dependency Status](https://gemnasium.com/badges/github.com/nidhinmahesh/susi_server.svg)](https://gemnasium.com/github.com/nidhinmahesh/susi_server)
```
Comment (Abhi2424shek) [inline README.md]: Can't you use `fossasia/susi_server`

Comment (nidhinmahesh) [inline README.md]: sure!

**Comment by mariobehling** [comment]
Please resolve conflicts and add issue number. Thanks.

**Comment by mariobehling** [comment]
Closing this due to inactivity. Please reopen if you actually plan to solve it. It is a tiny thing, that should not take so long.

**Pull request closed without merging**